beta/project-griffon/set-up-project-griffon.md

```diff
@@ -68,12 +74,12 @@ Registering the extension with Core, sends Experience Platform SDK events to an
 1. Register the extension wherever you are registering other extensions.
```

Comment:
Please rewrite this sentence as follows:
"Register the extension when you register other extensions."